script/release

```diff
@@ -326,7 +326,7 @@ def main():
 
     # push package to pypi
     step("Generate and push package to {}".format(args.repository or args.repository_url or "pypi"))
-    execute(["python", "setup.py", "bdist_wheel"], log=True)
+    execute(["python", "-m", "build", "--wheel"], log=True)
```

Comment:
is --wheel necessary, and also who install the [necessary package](https://pypi.org/project/build/) ?

Reply:
in general, we want to push both the wheel and the tar.gz, even if in this case, the package being a pure python file, the wheel could be enough, but it's considered a good practice to push both, afaik